src/liballoc/str.rs

```diff
@@ -37,6 +37,7 @@
 // Many of the usings in this module are only used in the test configuration.
 // It's cleaner to just turn off the unused_imports warning than to fix them.
 #![allow(unused_imports)]
+#![feature(fn_must_use)]
```

Comment:
Hm. I hoped this would get rid of
````
[00:24:20] warning: `#[must_use]` on methods is experimental (see issue #43302)
[00:24:20]    --> liballoc/str.rs:211:5
[00:24:20]     |
[00:24:20] 211 |     #[must_use]
[00:24:20]     |     ^^^^^^^^^^^
[00:24:20]     |
[00:24:20]     = help: add #![feature(fn_must_use)] to the crate attributes to enable
[00:24:20]
[00:24:20] warning: `#[must_use]` on methods is experimental (see issue #43302)
[00:24:20]    --> liballoc/str.rs:251:5
[00:24:20]     |
[00:24:20] 251 |     #[must_use]
[00:24:20]     |     ^^^^^^^^^^^
[00:24:20]     |
[00:24:20]     = help: add #![feature(fn_must_use)] to the crate attributes to enable
[00:24:20]
````
but apparently it does't. What am I doing wrong?

Reply:
You need to add `#![feature(fn_must_use)]` to src/liballoc/lib.rs instead.